src/__tests__/msw/mockData/singleUserContests.json

```diff
@@ -0,0 +1,16 @@
+{
+  "data": {
+    "userContestRanking": null,
+    "userContestRankingHistory": [
+      {
+        "attended": false,
+        "rating": 1500,
+        "ranking": 0,
+        "trendDirection": "NONE",
+        "problemsSolved": 0,
+        "totalProblems": 3,
+        "finishTimeInSeconds": 0
+      }
+    ]
+  }
+}
```

Comment:
Heres what I mean - my fetch brings back userContestRanking = null. Because I've never done a contest

Reply:
Thanks for this error correction 😌